Commons/bukkit/src/main/java/tc/oc/commons/bukkit/commands/MiscCommands.java

```diff
@@ -60,34 +60,18 @@
     public void listPlayerVersions(final CommandContext args, final CommandSender sender) throws CommandException {
         Audience audience = audiences.get(sender);
         if (args.hasFlag('a')) {
-            if (args.hasFlag('d')) {
-                Map<Integer, Integer> playerCountVersionMap = new HashMap<>();
-                userStore.stream().forEach(player -> {
-                    int version = player.getProtocolVersion();
-                    playerCountVersionMap.put(version, playerCountVersionMap.getOrDefault(version, 0) + 1);
-                });
-
-                audience.sendMessage(new HeaderComponent(new Component(ChatColor.AQUA).translate("list.player.versions.title")));
-                for (Map.Entry<Integer, Integer> entry : playerCountVersionMap.entrySet()) {
-                    audience.sendMessage(new TranslatableComponent("list.player.versions.message." + (entry.getValue() == 1 ? "singular" : "plural"),
-                            ChatColor.AQUA + entry.getValue().toString(),
-                            ChatColor.AQUA + MinecraftVersion.describeProtocol(entry.getKey()),
-                            String.format("%.1f", 100 * entry.getValue() / (double) userStore.count()) + "%"));
-                }
-            } else {
-                Map<String, Integer> playerCountVersionMap = new HashMap<>();
-                userStore.stream().forEach(player -> {
-                    int version = player.getProtocolVersion();
-                    playerCountVersionMap.put(MinecraftVersion.describeProtocolSimplified(version), playerCountVersionMap.getOrDefault(version, 0) + 1);
-                });
-
-                audience.sendMessage(new HeaderComponent(new Component(ChatColor.AQUA).translate("list.player.versions.title")));
-                for (Map.Entry<String, Integer> entry : playerCountVersionMap.entrySet()) {
-                    audience.sendMessage(new TranslatableComponent("list.player.versions.message." + (entry.getValue() == 1 ? "singular" : "plural"),
-                            ChatColor.AQUA + entry.getValue().toString(),
-                            ChatColor.AQUA + entry.getKey(),
-                            String.format("%.1f", 100 * entry.getValue() / (double) userStore.count()) + "%"));
-                }
+            Map<String, Integer> playerCountVersionMap = new HashMap<>();
+            userStore.stream().forEach(player -> {
+                String version = (args.hasFlag('d') ? MinecraftVersion.describeProtocol(player.getProtocolVersion()) : MinecraftVersion.describeProtocolSimplified(player.getProtocolVersion()));
```

Comment:
should probably make a `describeProtocol(int proto, boolean simple)` method in MinecraftVersion instead